src/controllers/cardsController.ts

```diff
@@ -1,80 +0,0 @@
-import * as cardServices from "../services/cardServices.js";
-import { Request, Response } from "express";
-
-
-
-export async function createNewCard(req: Request, res: Response) {
-
-  const { employeeId, cardType } = req.body
-
-  const card = await cardServices.createNewCard(employeeId, cardType)
-
-  res.status(201).send(card)
-
-}
-
-export async function activateCard(req: Request, res: Response) {
-
-  const { id } = req.params
-
-  const { securityCode, password } = req.body
-
-  const cardId: number = parseInt(id)
```

Comment:
É bacana deixar trechos de código mais relacionados juntos. Além disso, só é preciso tipar uma variável se o tipo não for claro pro ts (any). De modo que trechos como estes ficariam interessantes assim: 

```js
 const { id } = req.params;
 const cardId = parseInt(id);
 
 const { securityCode, password } = req.body
```